src/main/java/net/neoforged/neoforge/common/Tags.java

```diff
@@ -436,6 +436,10 @@ public static class Items {
          * Examples are Rotten Flesh's Hunger or Pufferfish's Nausea, or Poisonous Potato's Poison.
          */
         public static final TagKey<Item> FOODS_FOOD_POISONING = tag("foods/food_poisoning");
+        /**
+         * All foods edible by animals excluding poisonous foods.
```

Comment:
Maybe add ` (Does not include {@link ItemTags#PARROT_POISONOUS_FOOD})` to the end here? Just so that modders know what was excluded